apps/www/pages/angebote.js

```diff
@@ -89,4 +89,4 @@ class PledgePage extends Component {
   }
 }
 
-export default withDefaultSSR(withRouter(PledgePage))
+export default withRouter(PledgePage)
```

Comment:
please re-add

Reply:
Alright, this will however cause Apple-Pay to not be initially available on the first-render.